packages/react/src/drafts/TabPanels/TabPanels.docs.json

```diff
@@ -0,0 +1,72 @@
+{
+  "id": "tab_panels",
+  "name": "TabPanels",
+  "status": "draft",
+  "a11yReviewed": false,
+  "stories": [],
+  "props": [
+    {
+      "name": "aria-label",
```

Comment:
Could we could make `aria-label` (and I guess `aria-labelledby`) required in the types for `TabPanels` so that one of these must be set?

Reply:
I added `aria-labelledby` and throw an error if one of them isn't set. Let me know if there's a better way to do this!

Reply:
@owenniblock what would you think of something like this at the type level that could be a part of the component props? Was wondering if this would work or if there was a case that this wouldn't cover 👀 

```ts
type Label = {
  'aria-label': string;
  'aria-labelledby'?: never;
}

type Labelledby = {
  'aria-label'?: never;
  'aria-labelledby': string;
}

type Labelled = Label | Labelledby;
```